src/resources/time-slot.ts

```diff
@@ -31,6 +32,10 @@ export interface TimeSlotParameters {
   visibility?: number;
 }
 
+export interface TimeSlotMeta {
+  [key: string]: unknown;
```

Comment:
Just going to echo the previous comments about the `unknown` type being replaced by a union type.